### PR TITLE
Added new script for saving dumps

### PR DESCRIPTION
**Pull request by lexfor**
Added new script for copy segments.scdb and set for name current date and time. 
***
Default dump path : _/ostis-web-platform/kb.bin/dumps_. 
To set new dump path write new path in sc-web.ini file in segment "Repo", after line "Path".
***
First argument(optional) path to sc-web.ini.
 Default : _../config/sc-web.ini_

### CODE REVIEW

File: scripts/dump_saver.bash [hunk 1]
```diff
@@ -0,0 +1,52 @@
+echo "programm start..."
```
Comment (ShunkevichDV) [inline scripts/dump_saver.bash:1]: program

File: scripts/dump_saver.bash [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+isEnd=false
+isStart=false
+if [ -z $1 ]; then
+    file="../config/sc-web.ini"
+else
+    file=$1
+fi
+echo "sc-web.ini path: "$file
```
Comment (ShunkevichDV) [inline scripts/dump_saver.bash:9]: Filename may be different

**Review by ShunkevichDV — APPROVED** [review]
Need testing in real environment

File: scripts/dump_saver.bash (hunk 1 above)
Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:1]: Rename script extension to `.sh`

Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:1]: Add `#!/bin/bash` as a first line

Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:1]: Rename message as it's not a program. Smth like "script for saving dumps starts.."

Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:1]: Rename message as it's not a program. Smth like "script for saving dumps starts.."

File: scripts/dump_saver.bash [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  if [[ "$line" == "[Repo]" ]]
+  then
+    isStart=true
+  fi
+done < $file
+echo "segments.scdb path: "$path
+echo "dumps path: "$dumpsPath
+currentName="segments.scdb"
+LTIME=`stat -c %Z $path/$currentName`
+echo "wait new dump"
```
Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:34]:
```suggestion
echo "waiting for a new dump"
```

File: scripts/dump_saver.bash [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+   then    
+        if [ -d $dumpsPath ]; then
+            cp $path/$currentName $dumpsPath/$dumpdate.scdb
+        else 
+            mkdir $dumpsPath
+            cp $path/$currentName $dumpsPath/$dumpdate.scdb
+        fi
+       LTIME=$ATIME
+       echo "dump saved"
+       echo "wait new dump"
```
Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:49]:
```suggestion
       echo "waiting for a new dump"
```

File: scripts/dump_saver.bash [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+   fi
+   sleep 5
```
Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:51]: Could you please explain why we need `sleep` here and why 5 exactly

File: scripts/dump_saver.bash [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+done
```
Comment (MikhailSadovsky) [inline scripts/dump_saver.bash:52]:
```suggestion
done

```

**Review by MikhailSadovsky — CHANGES_REQUESTED** [review]

**Review by MikhailSadovsky — APPROVED** [review]

**Pull request merged**